Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_4.h

```diff
@@ -78,21 +78,19 @@
   #else
     #define Z_MIN_PIN      P1_00   // PWRDET
   #endif
-#else
-  #ifndef Z_STOP_PIN
-    #define Z_STOP_PIN     P1_27   // Z-STOP
-  #endif
+#elif ENABLED (BLTOUCH)
+  #define Z_STOP_PIN       P1_27   // Z-STOP
+    #else 
+      #define Z_STOP_PIN       P0_10   // Z-STOP
 #endif
 
 //
 // Z Probe (when not Z_MIN_PIN)
 //
-#ifndef Z_MIN_PROBE_PIN
-  #if Z_STOP_PIN != P1_27
+#if ENABLED (BLTOUCH)
     #define Z_MIN_PROBE_PIN P1_27
   #else
     #define Z_MIN_PROBE_PIN P0_10
-  #endif
 #endif
```

Comment:
So if BLtouch is enabled both Z_STOP_PIN and Z_MIN_PROBE_PIN share the same pin? and maybe DISABLED would make more sense at least to me because my bltouch is plugged into the probe connector that is P0_10